### PR TITLE
Add Tensorflow square distance calculation function 

### PR DESCRIPTION
**Pull request by GeorgeLiang3**
add the first square distance calculation function followed Miguel's theano version

### CODE REVIEW

File: gempy/core/interp_methods/tf_2D.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import tensorflow as tf
```
Comment (Leguark) [inline gempy/core/interp_methods/tf_2D.py]: Module level docstring missing

File: test/test_core/test_tensorflow.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+
+def test_basic_tf():
+    dips_position = tf.constant([[1,0],[1,1]])
+    surface_position = tf.constant([[[0,0],[2,0]],[[0,2],[2,2]]])
+    dips = tf.constant([[1,1],[1,1]])
+
+    test_model = Tensorflow_kriging_2D(dips_position,surface_position,dips)
+
+    # test_model.squared_euclidean_distance(surface_position,surface_position)
```
Comment (Leguark) [inline test/test_core/test_tensorflow.py]: removeee!

File: test/test_core/test_tensorflow.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+from gempy.core.interp_methods.tf_2D import Tensorflow_kriging_2D
+import tensorflow as tf
+
+
```
Comment (Leguark) [inline test/test_core/test_tensorflow.py]: pep 8

**Comment by alexanderzimmerman** [comment]
I changed the name of the PR as an example to make it more descriptive. @GeorgeLiang3, please update it if there is a better name. Thanks!

File: test/test_core/test_tensorflow.py (hunk 3 above)
Comment (GeorgeLiang3) [reply test/test_core/test_tensorflow.py]: What does this mean specifically

**Comment by pytzcarraldo** [comment]
@Leguark 
what exactly do you need me to do here? 

@GeorgeLiang3 
From this pull request content I have no real idea of what state this is in - is this still a work in progress? If so add [WIP] tag. Or does this need review?

**Comment by GeorgeLiang3** [comment]
> @Leguark
> what exactly do you need me to do here?
> 
> @GeorgeLiang3
> From this pull request content I have no real idea of what state this is in - is this still a work in progress? If so add [WIP] tag. Or does this need review?

Maybe I should close this for now? I was just trying my first pull request

**Comment by pytzcarraldo** [comment]
Hehe, no. I just wanted to know whats next for this pull request. It's fine to have a work in progress PR. I just want to encourage some more description to what is currently going on with it, as I don't now if this can be merged, or needs more work, or if you need help?

**Comment by alexanderzimmerman** [comment]
Maybe the purpose of this PR would make more sense if it was linked to Epic #274 and if there was some kind of roadmap for that Epic. 

Alternatively (or additionally), the purpose of this new code could be clarified in its test code.

**Comment by Leguark** [comment]
hey @GeorgeLiang3 I just saw that you have in the pull request hidden files of VS code (@pytzcarraldo can you double), I think something it is not right in your environment. Also your branch is getting really far from master. Maybe is better if you clone fresh from master and you add your code there?

**Pull request closed without merging**